Module.yaml

```diff
@@ -0,0 +1,23 @@
+version: 1.0.0
+name: Generic Translator Module
+description: Module translates commands from application to device compatible commands
```

Comment:
What type of device? Can it be used on all devices or only LoRa? It should be mentioned. If it is a specific device like LoRa then also name of the module should be updated to "Generic Translator LoRa"

Reply:
the idea behind this module is that will run standalone and translate commands from various devices in the future, so currently now it's only McClimate Vicki thermostat, later we can add more, and it would probably read it from DB. so we can't actually list all supported devices at this moment.